### DISCUSSION TITLE
[Feature] Add 'Hide person' option to photo Info view

### DISCUSSION THREAD

**Discussion by Yetangitu** [immich-app/immich#7343] [Feature Request]
### The feature

I often end up middle-mouse-button opening people from within a photo's info view only to be able to hide those persons from view because it becomes clear from within the context of that photo that those persons are not relevant. A good example is the following:

Open a 'selfie' type photo taken by some family members who are at a concert. You're only interested in their faces but the face detection algorithm detects a few dozens more which you neither know nor care about. Since the thumbnails often end up being low-resolution this only becomes clear when you view the info for that photo which shows the persons in context.

_Implementation_: Add a ´hide' option to the PEOPLE section of the Info view for photos which contain detected persons. This can either be a 'hide' badge on the thumbnails or a 'hide person' section similar to the 'pencil' edit section currently available. The 'person' should be highlighted in the photo when hovering the pointer over the thumbnails to make it clear which individual is targeted.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by sbkg0002** [comment]
Would love this. I toke screenshots of games in the past and some GTA characters are now part of my list of people ☺️

**Comment by Londoneye02** [comment]
+1

This would mean being much more productive in this cases

**Comment by bo0tzz** [comment]
#5135
